tests/update/updates_packagekit_kde.pm

```diff
@@ -51,7 +51,13 @@ sub run {
             # the assert_screen, record a soft failure
             wait_still_screen;
             if (match_has_tag('updates_none')) {
-                if (check_screen 'updates_none', 30) {
+                #Check if the applet reports that a restart is needed.
+                if (match_has_tag('updates_installed-restart')) {
+                    power_action 'reboot', textmode => 1;
+                    $self->wait_boot;
+                    setup_system;
+                }
+                elsif ('updates_none', 30) {
```

Comment:
Is that some new perl syntax or a missing function call?

Reply:
It's a function and since in Perl parentheses are optional for function calls and the rest of the module does not use them, I kept them as is.

Reply:
How does it now that it's supposed to call `check_screen`?

Reply:
Oh sorry I was looking at the wrong line.
Never review code after 00.00